src/hearings/components/parties-unavailable-dates/parties-unavailable-dates.component.html

```diff
@@ -3,6 +3,9 @@
     <span class="govuk-details__summary-text">See dates the parties will not be available</span>
   </summary>
   <div class="govuk-details__text">
+    <div *ngIf="unAvailableDates?.length === 0">
```

Comment:
can we do !unAvailableDates?.length ?

Reply:
seems they are doing same thing but I think the unAvailableDates?.length === 0 is more readable.